bcp47/bcp47.cabal

```diff
@@ -75,6 +75,7 @@ library
     , generic-arbitrary
     , iso639
     , megaparsec
+    , serialise
```

Comment:
Is this considered a heavy dependency? I wonder if `bcp47-orphans` is a better home. That seems to be our approach to inessential instances in this project.